### PR TITLE
Update documentation wsl shutdown command issue1977 

### LINKED ISSUE #1977: Documentation bug in "using-wsl2.md"
Sorry, not really a bug, but the best label fit available.

Documentation bug in kind/site/content/docs/user/using-wsl2.md
(https://github.com/kubernetes-sigs/kind/blob/master/site/content/docs/user/using-wsl2.md)

At the bottom of the document in the paragraph "Helpful Tips for WSL2"
Your proposed approach "wsl distro --shutdown" throws an error: "/bin/bash: distro: command not found".

<strike>You shutdown a distro with 
`wsl --shutdown <distro>`</strike>
<strike>i.e. the parameter order needs to be reversed.</strike>
This is incorrect, see [my comment below](https://github.com/kubernetes-sigs/kind/issues/1977#issuecomment-752272341)

### PR DESCRIPTION
**Pull request by gunadhya**
Fixes #1977 Documentation fix in [using-wsl2.md](https://github.com/kubernetes-sigs/kind/blob/master/site/content/docs/user/using-wsl2.md)
**Before**:
`wsl <distro> --shutdown`
**After**:
`wsl --shutdown <distro>`

### CODE REVIEW

**Comment by gunadhya** [comment]
I signed it

**Comment by gunadhya** [comment]
https://deploy-preview-1994--k8s-kind.netlify.app/docs/user/using-wsl2/

**Comment by aojea** [comment]
hmm  this says that `shutdown` doesn't accept any arguments
https://github.com/microsoft/WSL/issues/4539

**Comment by gunadhya** [comment]
image

@aojea I tried it on my system and seems to work. I'm using wsl2.

**Comment by aojea** [comment]
cool, thanks
/lgtm
/ok-to-test
/assign @BenTheElder

**Comment by aojea** [comment]
/hold

should it be terminate? 
https://github.com/kubernetes-sigs/kind/issues/1977#issuecomment-752272341

**Comment by gunadhya** [comment]
I've replaced shutdown command with terminate.

File: site/content/docs/user/using-wsl2.md [hunk 1]
```diff
@@ -86,6 +86,6 @@ nodes:
 
 ## Helpful Tips for WSL2
 
-- If you want to shutdown the WSL2 instance to save memory or "reboot", open an admin PowerShell prompt and run `wsl --shutdown <distro>`. Closing a WSL2 window doesn't shut it down automatically.
+- If you want to terminate the WSL2 instance to save memory or "reboot", open an admin PowerShell prompt and run `wsl -t <distro>`. Closing a WSL2 window doesn't shut it down automatically.
```
Comment (aojea) [inline site/content/docs/user/using-wsl2.md:89]: I prefer the long flag format for documentation purposes

**Comment by aojea** [comment]
Squash the commits please, we only need one commit for this change

**Comment by aojea** [comment]
/hold cancel
/lgtm
thanks

**Pull request merged**